### PR TITLE
Parse packageName and packageVersion from command line

### PR DESCRIPTION
**Pull request by daschult**

### CODE REVIEW

**Comment by olydis** [comment]
# 🤖 AutoRest automatic publish job 🤖
~~~ Haskell
> will publish once PR gets merged
~~~

**Comment by olydis** [comment]
# 🤖 AutoRest automatic feature coverage report 🤖
*feature set version 2.4.2*

## ❌️ General: 0%

no tests were run for this category


## ✔️ Azure: 100%

**Comment by olydis** [comment]
# 🤖 AutoRest automatic feature coverage report 🤖
*feature set version 2.4.2*

## ❌️ General: 0%

no tests were run for this category


## ✔️ Azure: 100%

File: src/Program.cs [hunk 1]
```diff
@@ -88,7 +88,20 @@ private T GetXmsCodeGenSetting<T>(CodeModel codeModel, string name)
             foreach (PropertyInfo propertyInfo in typeof(GeneratorSettingsJs).GetProperties())
             {
                 string propertyName = propertyInfo.Name;
-                Settings.Instance.CustomSettings[propertyName] = await GetValue<bool?>(propertyName.ToKebabCase()) ?? false;
+                string kebabCasePropertyName = propertyName.ToKebabCase();
```
Comment (RikkiGibson) [inline src/Program.cs]: Did you write ToKebabCase or was it already a thing in AutoRest?

Comment (daschult) [reply src/Program.cs]: I wrote it. It seemed pretty simple.

**Pull request merged**